docs/contributing/how-to-contribute-to-docs.md

```diff
@@ -94,8 +94,22 @@ Every PR is building website and on success it shows the link to preview.
 
 We use [Netlify](https://www.netlify.com/) for hosting. We are using Open Source license (PRO). Thanks Netlify for this!
 
-On every commit to `master` netlify runs CI that invokes `make web` (defined in [netlify.toml](https://github.com/improbable-eng/thanos/blob/master/netlify.toml)))
+On every commit to `master` netlify runs CI that invokes `make web` (defined in [netlify.toml](/netlify.toml))
 
 NOTE: Check for status badge in README for build status on the page.
 
 If master build for netlify succeed, the new content is published automatically.
+
+## Test locally
```

Comment:
I think this makes sense to have, but can we somehow integrate this into `Testing` section above? (: